### PR TITLE
add bulk-lgtm tool.

### PR DESCRIPTION
**Pull request by RobinsChens**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1699)

### CODE REVIEW

File: mungegithub/mungers/bulk-lgtm.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import (
+    "encoding/json"
+    "net/http"
+
+    "k8s.io/contrib/mungegithub/github"
+    "k8s.io/contrib/mungegithub/features"
+
+    "github.com/golang/glog"
+	githubapi "github.com/google/go-github/github"
+	"github.com/spf13/cobra"
```
Comment (eparis) [inline mungegithub/mungers/bulk-lgtm.go]: whole PR needs to be `go fmt`

File: mungegithub/mungers/bulk-lgtm.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+)
+
+var _ Munger = &BulkLGTM{}
```
Comment (eparis) [inline mungegithub/mungers/bulk-lgtm.go]: I assume this is a placeholder until you do the RegisterOrDie() call?

File: mungegithub/mungers/bulk-lgtm.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+// Bulk LGTM knows how to aggregate a large number of small PRs into a single page for
+// easy bulk review.
+type BulkLGTM struct {
+    config *github.Config
```
Comment (eparis) [inline mungegithub/mungers/bulk-lgtm.go]: Why do you need this?

File: mungegithub/mungers/bulk-lgtm.go [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+func (b *BulkLGTM) ServePRs(res http.ResponseWriter, req *http.Request) {
+    var data []byte
+    var err error
+    if b.currentPRList == nil {
+        data = []byte("[]");
+    }
+    data, err = json.Marshal(b.currentPRList)
+    if err != nil {
+        res.Header().Set("Content-type", "text/plain")
```
Comment (eparis) [inline mungegithub/mungers/bulk-lgtm.go]: Why are you only setting the Content-type on error?

File: mungegithub/mungers/bulk-lgtm.go [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        res.WriteHeader(http.StatusInternalServerError)
+        return
+    }
+    
+	res.WriteHeader(http.StatusOK)
+    res.Write(data)
```
Comment (eparis) [inline mungegithub/mungers/bulk-lgtm.go]: Is there some hope to make a reasonable web front end to this json blob?

**Comment by test-foxish** [comment]
recomputing cla status...

**Comment by grodrigues3** [comment]
[APPROVALNOTIFIER] The Following OWNERS Files Need Approval:

- **mungegithub**

We suggest the following people:
cc @eparis 
 You can indicate your approval by writing `/approve` in a comment
 You can cancel your approval by writing `/approve cancel`in a comment

**Comment by foxish** [comment]
Closing this. Please reopen if there's further work here.

**Pull request closed without merging**